common/lib/xmodule/xmodule/assetstore/assetmgr.py

```diff
@@ -49,14 +50,9 @@ class AssetManager(object):
     @contract(asset_key='AssetKey', throw_on_not_found='bool', as_stream='bool')
     def find(asset_key, throw_on_not_found=True, as_stream=False):
         """
-        Finds a course asset either in the assetstore -or- in the deprecated contentstore.
+        Gets course asset in the deprecated contentstore.
+        This method was previously searching course asset either in the assetstore -or- in the deprecated contentstore.
```

Comment:
I'd add that we were able to not look for it in the assetstore because we would throw an error if we actually found it!